config.toml

```diff
@@ -171,6 +171,11 @@ githubbranch = "v1.1-branch"
   githubbranch = "v0.7-branch"
   url = "https://v0-7.kubeflow.org"
 
+[[params.versions]]
+  version = "v1.0"
+  githubbranch = "v1.0-branch"
+  url = "https://v1-0-branch.kubeflow.org"
```

Comment:
Why do we use `v1-0-branch` as url prefix? It seems previous versions were using `v0-7.kubeflow.org`

Reply:
I think @jlewi meant https://v1-0-branch.kubeflow.org is available even before you set up this config, but this if we follow previous conventions, we can make https://v1-0.kubeflow.org also available.

Please correct me if there's something I missed.

Reply:
@Bobgy Let's make `v1-0.kubeflow.org` available - do you know how? I'm looking at https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#version-the-website. Thank you!

(fyi I linked v1.0 to `https://v1-0-branch.kubeflow.org` because it was the only URL that worked and fixing `config.toml` seemed urgent - some folks were looking for the v1.0 docs.) cc @jlewi